src/components/Calendar/src/Calendar.vue

```diff
@@ -0,0 +1,455 @@
+<template>
+	<div
+		:class="$s.Calendar"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div :class="$s.CalendarHeader">
+			<m-button
+				:disabled="shouldDisableHeaderButton(-1)"
+				:color="headerButtonColor"
+				size="small"
+				variant="primary"
+				tabindex="-1"
+				@click="incMonth(-1)"
+			>
+				<chevron-left
+					:class="$s.CalendarHeaderButtonIcon"
+				/>
+			</m-button>
+
+			<span :class="$s.CalendarHeaderTitle">
+				{{ monthName }}
+			</span>
+
+			<m-button
+				:disabled="shouldDisableHeaderButton(1)"
+				size="small"
+				variant="primary"
+				tabindex="-1"
+				color="#f2f2f2"
+				@click="incMonth(1)"
+			>
+				<chevron-right
+					:class="$s.CalendarHeaderButtonIcon"
+				/>
+			</m-button>
+		</div>
+		<table :class="$s.CalendarTable">
+			<thead>
+				<tr>
+					<th
+						v-for="day in weekdays"
+						:key="`day-${day}`"
+						:class="$s.DateHeaderCell"
+					>
+						{{ day }}
+					</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr
+					v-for="(week, wi) in weeks"
+					:key="`week-${wi}`"
+				>
+					<td
+						v-for="(date, di) in week"
+						:key="`date-${wi-di}`"
+						:class="$s.DateCell"
+						@click.prevent="emitDate(date)"
+					>
+						<button
+							v-if="date"
+							:class="[$s.DateCellButton, {
+								[$s.selected]: isDateSelected(date),
+								[$s.disabled]: isDateDisabled(date),
+								[$s.today]: isToday(date)
+							}]"
+							type="button"
+							tabindex="-1"
+						>
+							{{ date }}
+						</button>
+					</td>
+				</tr>
+			</tbody>
+		</table>
+	</div>
+</template>
+
+<script>
+import {
+	startOfWeek,
+	startOfMonth,
+	eachDayOfInterval,
+	getDay,
+	lastDayOfMonth,
+	addDays,
+	format,
+	isToday,
+	isAfter,
+	isSameDay,
+	isSameMonth,
+	isSameYear,
+	addMonths,
+	parse,
+} from 'date-fns';
+import * as dateFnsLocale from 'date-fns/locale';
```

Comment:
Can we get away with not bundling this in? This adds a lot and we previously didn't need it.

If not, you should exclude locales we don't support: https://date-fns.org/v2.19.0/docs/webpack